static/app/views/starfish/components/tableCells/spanIdCell.tsx

```diff
@@ -14,6 +17,7 @@ interface Props {
 }
 
 export function SpanIdCell({
+  moduleName,
```

Comment:
instead of adding a separate moduleName prop, consider inferring the module from path (`location.pathname`). helper function that maps paths to ModuleName enum